content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,236 @@
+
+# O﻿verview
+
+Let's start with a simple flowchart describing the 10000-foot view of the creation process:
+
+![](/img/wod-b-process.png "Workshop's creation flow.")
+
+A﻿s you can see, there's no rocket science here. Just common sense. Depending on the workshop you wish to create, some obvious requirements should show up. A workshop based on a programmatic language, for instance, may require the relevant kernel to be set up on the JupyterHub server. The following [page](https://gist.github.com/chronitis/682c4e0d9f663e85e3d87e97cd7d1624) lists all available kernels.
+
+S﻿ome other workshops might need a proper infrastructure to run on. A kubernetes101 workshop for instance could not exist without the presence of a proper Kubernetes cluster. The same thing goes for any HPE-related solutions.
+
+F﻿rom an infrastructure standpoint, a minimum number of environments are necessary. The project makes it easy to deploy a development and test environments, a staging environment, and at least one production environment. 
+
+Here are a few definitions of the different environments :
+
+1. **Development Environment:** This is where application/system development tasks, such as designing, programming, debugging of a workshop, etc., take place.
+2. **Test Environment:** As the name implies, this is where the workshop testing is conducted to find and fix errors.
+3. **Staging Environment:** Here, all the work done in the development environment is merged into the built system (often used to automate the process of software compilation) before it is moved into the production environment.
+4. **Production Environment:** The last environment in workshop development, this is where new builds/updates are moved into production for end users.
+
+The HPE Developer Community actually started with only a development, test and staging environments on one side and a production on the other side.
+
+In this post, I won't focus on the subject selection process. I'll leave that to you to figure it out. I will, however, talk a little bit again about the infrastructure, especially the dedicated scripts and variables that you need to create to support the lifecycle of the workshop. As usual, there are two sides to the workshop's creation. What should be done on the backend and what needs to be done on the api db server mainly.
+
+![](/img/wod-blogserie3-archi3.png "WOD Overview.")
+
+What is a workshop? What do you need to develop?
+
+L﻿et's imagine that you plan to create a new workshop on a topic on which you have knowledge to transfer.  Let's call him **Matt**. He was kind enough to agree with working with me on creating a new workshop. After our first meeting, where I explained the creation process, and the expectations, we were able to quickly start working together. 
+
+We defined what is needed:
```

Comment:
Change this to either:

We defined what was needed:

We defined what we needed:

("is" is in the present tense, but both "defined" and "needed" are in the past tense, so you can't combine them)